jest.config.js

```diff
@@ -0,0 +1,25 @@
+/*
+ * For a detailed explanation regarding each configuration property, visit:
+ * https://jestjs.io/docs/configuration
+ */
+
+module.exports = {
+  // All imported modules in your tests should be mocked automatically
```

Comment:
- It may look better to remove comments if they are not necessary

Reply:
Done ✅